api/run.sh

```diff
@@ -17,8 +17,8 @@
 #
 
 export ENV=local
-pwd=`pwd`
+pwd=$(pwd)
 
-cd ./output
+cd ./output || exit
```

Comment:
should give some message

Reply:
If `cd` fails, the system will give the error message `No such file or directory exists`

Reply:
And the script will exit after `cd` fails to execute, so there is no need to print some message